civitai-downloader-v2/tests/api/test_integration.py

```
@@ -0,0 +1,238 @@
+"""
+Integration tests for API endpoints.
+"""
+import pytest
```

Ruff (0.12.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)